selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/ImportProjectFromGitHubTest.java

```diff
@@ -0,0 +1,128 @@
+import org.eclipse.che.selenium.pageobject.dashboard.ProjectSourcePage;
+import org.eclipse.che.selenium.pageobject.dashboard.workspaces.Workspaces;
+import org.openqa.selenium.TimeoutException;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+public class ImportProjectFromGitHubTest {
+  private final String WORKSPACE = NameGenerator.generate("ImtMvnPrjGitHub", 4);
+  private static final String GITHUB_PROJECT_NAME = "AngularJS";
+
+  private String projectName;
+  private String ideWin;
+
+  @Inject
+  @Named("github.username")
+  private String gitHubUsername;
+
+  @Inject
+  @Named("github.password")
+  private String gitHubPassword;
+
+  @Inject private Loader loader;
+  @Inject private GitHub gitHub;
+  @Inject private Dashboard dashboard;
+  @Inject private Workspaces workspaces;
+  @Inject private TestUser defaultTestUser;
+  @Inject private ProjectExplorer projectExplorer;
+  @Inject private CreateWorkspace createWorkspace;
+  @Inject private ProjectSourcePage projectSourcePage;
+  @Inject private SeleniumWebDriver seleniumWebDriver;
+  @Inject private TestWorkspaceServiceClient workspaceServiceClient;
+
+  @BeforeClass
+  public void setUp() {
+    projectName = gitHubUsername + "-" + GITHUB_PROJECT_NAME;
+
+    dashboard.open();
+  }
+
+  @AfterClass
+  public void tearDown() throws Exception {
+    workspaceServiceClient.delete(WORKSPACE, defaultTestUser.getName());
+  }
+
+  @Test
+  public void checkAbilityImportMavenProjectTest() throws Exception {
+    ideWin = seleniumWebDriver.getWindowHandle();
+
+    dashboard.waitDashboardToolbarTitle();
+    dashboard.selectWorkspacesItemOnDashboard();
+    workspaces.clickOnNewWorkspaceBtn();
+    createWorkspace.waitToolbar();
+    createWorkspace.selectStack(JAVA.getId());
+    createWorkspace.typeWorkspaceName(WORKSPACE);
+
+    projectSourcePage.clickOnAddOrImportProjectButton();
+    projectSourcePage.selectSourceTab(GITHUB);
+    projectSourcePage.clickOnConnectGithubAccountButton();
+    clickOnConnectYourGithubAccountButton();
+    projectSourcePage.waitGithubProjectsList();
+    projectSourcePage.selectProjectFromList(GITHUB_PROJECT_NAME);
+    projectSourcePage.clickOnAddProjectButton();
+    createWorkspace.clickOnCreateWorkspaceButton();
+
+    try {
+      seleniumWebDriver.switchFromDashboardIframeToIde(ELEMENT_TIMEOUT_SEC);
+    } catch (TimeoutException ex) {
+      // Remove try-catch block after issue has been resolved
+      fail("Known issue https://github.com/eclipse/che/issues/6323");
+    }
+
+    projectExplorer.waitProjectExplorer();
+    projectExplorer.waitItem(projectName);
+    projectExplorer.waitFolderDefinedTypeOfFolderByPath(projectName, PROJECT_FOLDER);
+  }
+
+  private void clickOnConnectYourGithubAccountButton() {
+    seleniumWebDriver.switchToNoneCurrentWindow(ideWin);
+
+    gitHub.waitAuthorizationPageOpened();
```

Comment:
This page could never be opened. Did you try to run the test in multiuser mode after some other test which required oauth authorization with github (for example, **ImportRecursiveSubmoduleTest**) had been executed?